features/support/touch_helpers.rb

```diff
@@ -0,0 +1,37 @@
+module Calabash
```

Comment:
Hey, this looks like third-party code, it's usually good practice to document where it comes from incase there are problems, or whatnot

Reply:
I also thought multi-touch existed in calabash already, if it does, doing this may be confusing in the future for various reasons (it's called shadowing, and when you expect X to behave like X, but it doesn't, it can be confusing.  if it does exist already, but you would like different behaviour, name it differently perhaps?

Reply:
You are absolutely right, I'm going to remove the "touch_helpers.rb". I used this only during my testing and didn't delete it. My bad!